src/utils/error.ts

```diff
@@ -17,12 +22,39 @@ export abstract class CliKintoneError extends Error {
   protected _toStringCause(): string {
     if (this.cause instanceof CliKintoneError) {
       return this.cause.toString();
+    } else if (this.cause instanceof KintoneAllRecordsError) {
+      return this._toStringKintoneAllRecordsError(this.cause);
+    } else if (this.cause instanceof KintoneRestAPIError) {
+      return this._toStringKintoneRestAPIError(this.cause);
     }
     return this.cause + "\n";
   }
 
+  protected _toStringKintoneAllRecordsError(
```

Comment:
Make it private
```suggestion
  private _toStringKintoneAllRecordsError(
```

Reply:
Fixed with a557a9d367ac6093128396a4c381b0ca5a23a5bd